netsim/ansible/templates/initial/cumulus_nvue.j2

```diff
@@ -50,21 +59,49 @@
           address:
             dhcp: {}
         type: eth
+{% for l in interfaces|default([]) if l.type!='loopback' %}
+{{      decl_interface(l) }}
+{% endfor %}
+
+{% for lb in netlab_interfaces if lb.type=='loopback' %}
+- set:
+{%   if lb.vrf is not defined %}
+    interface:
       lo:
+        type: loopback
+{%     if lb.mtu is defined %}
+        link:
+          mtu: {{ lb.mtu }}
+{%     endif %}
         ip:
           address:
-{% if 'ipv4' in loopback %}
-            {{ loopback.ipv4 }}: {}
-{% else %}
-            127.0.0.1: {}
-{% endif %}
-{% if 'ipv6' in loopback %}
-            {{ loopback.ipv6 }}: {}
-{% else %}
+{%     if 'ipv4' in lb %}
+            {{ lb.ipv4 }}: {}
+{%     else %}
+            127.0.0.1/8: {}
+{%     endif %}
+{%     if 'ipv6' in lb %}
+            {{ lb.ipv6 }}: {}
+{%     else %}
           ipv6:
             enable: off
-{% endif %}
-        type: loopback
-{% for l in interfaces|default([]) if l.type in ['lan','p2p','stub','svi','lag'] %}
-{{      decl_interface(l) }}
+{%     endif %}
+{%   else %}
+    vrf:
+      {{ lb.vrf }}:
```

Comment:
It looks to me like this logic might incorrectly handle the scenarios where there are multiple loopbacks in a VRF. I know those should be rare, but we still have to do it right (not my fault that CL NVUE has a suboptimal data model ;)

Reply:
![image](https://github.com/user-attachments/assets/e41d6a99-a40f-4dfd-89ce-1186b5a9061c)

![image](https://github.com/user-attachments/assets/64b8c478-a671-4cd9-a7b5-c7c1ba5a5381)

Multiple ```set``` commands with different IPs are cumulative. Did find an issue with disabling ipv6 on vrf loopbacks - that's not needed